crates/sui-bridge-indexer/src/worker.rs

```diff
@@ -64,16 +73,19 @@ impl BridgeWorker {
         checkpoint: u64,
         timestamp_ms: u64,
     ) -> Result<Vec<TokenTransfer>> {
+        self.metrics.total_sui_bridge_transactions.inc();
         if let Some(events) = &tx.events {
             let token_transfers = events.data.iter().try_fold(vec![], |mut result, ev| {
-                if let Some(data) = Self::process_sui_event(ev, tx, checkpoint, timestamp_ms)? {
+                if let Some(data) =
+                    Self::process_sui_event(ev, tx, checkpoint, timestamp_ms, &self.metrics)?
+                {
                     result.push(data);
                 }
                 Ok::<_, anyhow::Error>(result)
             })?;
 
             if !token_transfers.is_empty() {
-                debug!(
+                info!(
```

Comment:
having individual event process log at info level can make the indexer quite chatty, is that ok?

Reply:
yeah we had that conversation and we decided to go with a more chatty interface for now. We can revisit if we end up with too much logging.
Thanks